dali/test/python/test_operator_arithmetic_ops.py

```diff
@@ -44,23 +44,26 @@ def list_product(*args):
 # A number used to test constant inputs
 magic_number = 7
 
-unary_input_kinds = ["cpu", "gpu", "cpu_scalar", "gpu_scalar", "cpu_scalar_legacy", "gpu_scalar_legacy"]
+unary_input_kinds = [
+    "cpu", "gpu", "cpu_scalar", "gpu_scalar", "cpu_scalar_legacy", "gpu_scalar_legacy"
+]
 
-all_input_kinds = ["cpu", "gpu", "cpu_scalar", "gpu_scalar", "cpu_scalar_legacy", "gpu_scalar_legacy",  "const"]
+all_input_kinds = [
+    "cpu", "gpu", "cpu_scalar", "gpu_scalar", "cpu_scalar_legacy", "gpu_scalar_legacy", "const"
+]
 
 # We cannot have 'Constant x Constant' operations with DALI op.
 # And scalar is still a represented by a Tensor, so 'Scalar x Constant' is the same
 # as 'Tensor x Constant'.
-bin_input_kinds = (
-        list_product(["cpu", "gpu"], all_input_kinds) +
-        list_product(["cpu_scalar", "gpu_scalar", "const"], ["cpu", "gpu"]))
+bin_input_kinds = (list_product(["cpu", "gpu"], all_input_kinds) +
+                   list_product(["cpu_scalar", "gpu_scalar", "const"], ["cpu", "gpu"]))
 
 ternary_input_kinds = list_product(all_input_kinds, all_input_kinds, all_input_kinds)
 ternary_input_kinds.remove(("const", "const", "const"))
 
-integer_types = [np.bool_,
-                 np.int8, np.int16, np.int32, np.int64,
-                 np.uint8, np.uint16, np.uint32, np.uint64]
+integer_types = [
+    np.bool_, np.int8, np.int16, np.int32, np.int64, np.uint8, np.uint16, np.uint32, np.uint64
+]
```

Comment:
and the nice grouping went to Hell... - I think the original formatting was actually correct under PEP8?